src/dotty/tools/dotc/repl/CompilingInterpreter.scala

```diff
@@ -6,7 +6,7 @@ import java.io.{
   File, PrintWriter, PrintStream, StringWriter, Writer, OutputStream,
   ByteArrayOutputStream => ByteOutputStream
 }
-import java.lang.{Class, ClassLoader}
+import java.lang.{Class, ClassLoader, Thread, System, StringBuffer}
```

Comment:
Yes, that sounds like a prime candidate for rewriting. I opened https://github.com/scalacenter/scalafix/issues/21